### ISSUE TITLE
I'd like to exclude the "Version: Crypt::OpenPGP" line in the armored output.

### ISSUE THREAD

**Issue by floatingatoll** [btrott/Crypt-OpenPGP#26]
```
  package Crypt::OpenPGP::Armour;

  sub armour {

      $headers->{Version} = Crypt::OpenPGP->version_string;
```

How can this line be bypassed, so that the resulting output contains no ```Version:``` string?

**Comment by floatingatoll** [comment]
See also [bugzilla.mozilla.org #1181406](https://bugzilla.mozilla.org/show_bug.cgi?id=1181406).

**Comment by sergeyromanov** [comment]
@floatingatoll I've just pushed a branch with NoVersion parameter -- 66f37fa6, could you please see if this will work for you. If yes, I'll update the docs and cut a release.

**Comment by floatingatoll** [comment]
That looks perfect. Thank you!

**Comment by sergeyromanov** [comment]
I've just shipped Crypt::OpenPGP 1.12 with this change.

You're welcome!
